docs/en/ingest-management/fleet/fleet-settings-output-kafka.asciidoc

```diff
@@ -0,0 +1,367 @@
+production environments is strongly discouraged.
+
+`Strict`::
+Verifies that the provided certificate is signed by a trusted
+authority (CA) and also verifies that the server's hostname (or IP address)
+matches the names identified within the certificate. If the Subject Alternative
+Name is empty, it returns an error.
+
+`Certificate`::
+Verifies that the provided certificate is signed by a
+trusted authority (CA), but does not perform any hostname verification.
+
+The default value is `Full`.
+
+|===
+
+[discrete]
+== Partitioning settings
+
+The number of partitions created is set automatically by the Kafka broker based on the list of topics. Records are then published to partitions either randomly, in round-robin order, or according to a calculated hash.
+
+[cols="2*<a"]
+|===
+
+|
+[id="kafka-output-partitioning-random"]
+**Random**
+
+| Publish records to Kafka output broker event partitions randomly. Specify the number of events to be published to the same partition before the partitioner selects a new partition.
+
+// =============================================================================
+
+|
+[id="kafka-output-partitioning-roundrobin"]
+**Round robin**
+
+| Publish records to Kafka output broker event partitions in a round-robin fashion. Specify the number of events to be published to the same partition before the partitioner selects a new partition.
+
+// =============================================================================
+
+|
+[id="kafka-output-partitioning-hash"]
+**Hash**
+
+| Publish records to Kafka output broker event partitions based on a hash computed from the specified list of fields. If a field is not specified, the Kafka event key value is used.
+
+|===
+
+[discrete]
+== Topics settings
+
+Use these options to dynamically set the Kafka topic for each {agent} event.
+
+[cols="2*<a"]
+|===
+
+|
+[id="kafka-output-topics-default"]
+**Default topic**
+
+| Set the default topic to use. Click **Add topic processor** to specify additional processors to set topics based on event contents.
+
+// =============================================================================
+
+|
+[id="kafka-output-topics-processors"]
+**Processors**
+
+| For each processor provide a condition, the event value to check against, and the resulting Kafka topic.
+
+Refer to <<processor-conditions,conditions>> in the {agent} processor syntax for condition descriptions. Currently the `equals`, `contains`, and `regexp` conditions are available.
+
+Events that don't match against any defined processor are set to the default topic.
+
+Processors are applied in the order that they appear, from top to bottom.
+
+As an example for setting up your processors, you might want to route log events based on severity. To do so, you can specify a default topic for all events not matched by other processors:
+
+* `%{[fields.log_topic]}`.
+
+Then, create a processor to route critical events:
+
+* Condition: `Contains`
+* Value: `message: “CRITICAL”`  
```

Comment:
It could be valuable to emphasize that the value must adhere to the format of '[key]: value', where both the key and value are strings. For instance: host.port: 2000, message: Hello